.travis.yml

```diff
@@ -9,8 +9,12 @@ script:
   - sudo apt update
   - sudo apt install --yes snapd
   - sudo snap install *.snap --classic --dangerous
+  - sleep 10
```

Comment:
Why?

Reply:
Removed. Waiting should be done inside the smoke-test script.